products/pubsub/api.yaml

```diff
@@ -27,10 +27,18 @@ apis_required:
 objects:
   - !ruby/object:Api::Resource
     name: 'Topic'
-    base_url: projects/{{project}}/topics
-    create_verb: :PUT
     description: |
       A named resource to which messages are sent by publishers.
+    references: !ruby/object:Api::Resource::ReferenceLinks
```

Comment:
Did anything actually change in this file besides reorderings?

Reply:
no - i originally had async in this file and was annoyed at the ordering of these fields. I moved async to Terraform once I realized that ansible doesn't support PollAsync at all and only started failing because it also uses pubsub. I can revert these changes if you'd like.